public/index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Open+Sans&display=swap" rel="stylesheet">
```

Comment:
I love Google Fonts 🤩 !

**Small suggestions for the future:**
- Choose two typographies: heading & body text.
- About heading typography, try to find typography with multiple styles (level of bold: `400 - 900`). Because it's a little difficult to distinguish between texts or bold text. Of course, they are only ideas.